pkgs/by-name/ca/cables/package.nix

```
@@ -0,0 +1,38 @@
+{ pkgs, lib, ... }:
+
+let
+  pname = "cables";
+  version = "0.3.2";
+  name = "${pname}-${version}";
+
+  src = pkgs.fetchurl {
+    url = "https://github.com/cables-gl/cables_electron/releases/download/v0.3.2/cables-0.3.2-linux-x64.AppImage";
+    sha256 = "sha256-D5Bgg5D03FPQF2HKow4yehcqToo4dpPudBg0W4UnqFs=";
+  };
+
+  appimageContents = pkgs.appimageTools.extract {
+    inherit pname version src;
+    postExtract = ''
+      substituteInPlace $out/${name}.desktop --replace 'Exec=AppRun' 'Exec=${pname}'
```

Although the package name happens to be the same as the executable name, we try not to use it in this way. They are not logically connected.

```suggestion
      substituteInPlace $out/${name}.desktop --replace 'Exec=AppRun' 'Exec=cables'
```